src/arch/arm/lib/debug/stack_iter.c

```diff
@@ -48,11 +48,10 @@ void stack_iter_current(stack_iter_t *f) {
 	 * by hand */
 
 	stack_iter_next(f);
-	f->fp = (void*) *((int*)f->fp - 3);
 }
 
 void *stack_iter_get_fp(stack_iter_t *f) {
-	return f->fp - 3;
+	return f->fp - 4;
```

Comment:
`stack_iter_get_fp` is not used, can't check correctness of this change

Reply:
🤔
Let's just delete this function :)

Reply:
It's trivial for x86 and sparc, so I think it's ok to remove it from these architectures as well